### ISSUE TITLE
Attachment Uploader Multi Files is not working in Phones

### ISSUE THREAD

**Issue by yeruva0795** [ramarao9/AttachmentUploader#27]
Hi,

I have installed managed solution in CRM instance for multi files selection feature where in Web Client it is working as expected, but in Phone the multi select files is **not working.**

**Comment by celer1tas** [comment]
Same issue here when rendering it in new Microsoft Field Service Mobile app.
Uploading single file works fine but when trying to select multiple files I am not allowed to perform the selection. 
I've tested on Samsung S21 with Android 12 using multiple apps: such as Gallery, My Files.

**Comment by ramarao9** [comment]
Sorry for the delay, will review the issue and get back to you.

**Comment by celer1tas** [comment]
Observation: when I try the same using this react-dropzone example website: https://react-dropzone-uploader.js.org/docs/examples,  the chrome browser on the same phone allows multiple file selection.
[](url)
![Screenshot_20220419-115829_Android System](https://user-images.githubusercontent.com/5678736/163968861-d8c768f9-15dc-4624-a2df-ee55cee8026d.jpg)

**Comment by ramarao9** [comment]
I was able to reproduce the behavior. I noticed that the mobile apps for Dynamics 365 in Android do not allow selecting more than one file at a time. While the PCF component can allow multiple selections, the file picker is managed by how the mobile app wants to control selections. 

I also tested this behavior using the Field Service app on iOS and was able to select multiple files when using the Photo Library option.

In addition, using the Chrome browser on a mobile app and accessing the Dynamics 365 environment will allow multiple selections, similar to the React DropZone test above.

In Summary, I don't think this is something we have control over, and selecting multiple files on the Dynamics 365 apps would need to be a feature that Microsoft needs to implement.

**Comment by bcatelin** [comment]
I have contacted MS support who confirmed that the field service mobile app does not support multiple file uploads. They told me to create a feature request, which I did.

**Comment by ramarao9** [comment]
@bcatelin That's cool, thank you!
